docs/en/02_Developer_Guides/00_Model/02_Relations.md

```diff
@@ -324,7 +324,7 @@ This is declared via array syntax, with the following keys on the many_many:
  - `from` Name of the has_one relationship pointing back at the object declaring many_many
  - `to` Name of the has_one relationship pointing to the object declaring belongs_many_many.
 
-Just like a any normal DataObject, you can apply a default sort which will be applied when
+Just like in any normal DataObject, you can apply a default sort which will be applied when
```

Comment:
```suggestion
Just like any normal DataObject, you can apply a default sort which will be applied when
```

I think `in` (or `a`) is actually just unnecessary here.

Reply:
Thanks for pointing this out!
My English is quite rusty, tbh 😅

Reply:
FWIW, `in` does make sense there too (English 🙉 ), but in the interest of keeping the English as simple as possible it's best to remove these words that don't _have_ to be there.